src/services/provider/geoResource.provider.js

```diff
@@ -152,6 +152,8 @@ export const loadBvvGeoResourceById = (id) => {
  * GPX,GEOJSON,EWKT: `{url}||[{label}]`
  *
  * WMS: `{url}||{layer}||[{label}]`
+ *
+ * OAF: `{url}||{collectionId}`
```

Comment:
does not match with implementation:
`
const collectionId = parts[1];
const label = parts[2];
`
expect something like 
`* OAF: `{url}||{collectionId}||label``